### ISSUE TITLE
[Not ready for Triage/Review Meeting][Win32][Accessibility] [A11yMAS][Screen Readers][MAS4.2.1][ Screen reader is not announcing the changed value in “>>> edit field” in console section.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#726]
**User Experience:**
Screen reader user will not get the information of the value present in the field if screen reader is not announcing the value.

**Test Environment:**
OS Build: 16299.19
Excel version:1712(8808.1000)
Tools Used: Narrator
**Prerequisites:**
Add script lab “Add ON” In Excel

**Attachments:**
[MAS4.2.1_Value is not announced.zip](https://github.com/OfficeDev/script-lab/files/1473652/MAS4.2.1_Value.is.not.announced.zip)

**Repro Steps:**
1.	Open Script Lab in Excel
2.	Navigate to Run button and activate it.
3.	Navigate to Run pane Using F6 key
4.	Navigate to “console” button  and activate it..
5.	Navigate to “>>> edit field” and verify screen reader is announcing the changed value inside that field when user press tab key.

**Current Behavior:**
Screen reader is not announcing the changed value in “>>> edit field” in console section.

**Expected Behavior:**
Screen reader should announce the changed the value so that user will get to know the information of changed value.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.2.1
